starter_code/queries.md

```diff
@@ -4,45 +4,51 @@
 
 ### 1. All the companies whose name match 'Babelgum'. Retrieve only their `name` field.
 
-<!-- Your Code Goes Here -->
+Filter :{name:"Babelgum"}
+Project:{name:1}
 
 ### 2. All the companies that have more than 5000 employees. Limit the search to 20 companies and sort them by **number of employees**.
 
-<!-- Your Code Goes Here -->
+Filter:{number_of_employees:{$gt:5000} }
+Sort :{number_of_employees:1}
+Limit: 20
 
 ### 3. All the companies founded between 2000 and 2005, both years included. Retrieve only the `name` and `founded_year` fields.
 
-<!-- Your Code Goes Here -->
+Filter:{founded_year: {$gte:2000, $lte:2005}}
+Project :{name:1,founded_year:1}
 
 ### 4. All the companies that had a Valuation Amount of more than 100.000.000 and have been founded before 2010. Retrieve only the `name` and `ipo` fields.
-
-<!-- Your Code Goes Here -->
+FIlter: {$and: [{"ipo.valuation_amount":{$gte:100000000}}, {founded_year:{$lt:2010}}]}
+Projet: {ipo:1,name:1}
 
 ### 5. All the companies that have less than 1000 employees and have been founded before 2005. Order them by the number of employees and limit the search to 10 companies.
-
-<!-- Your Code Goes Here -->
+Filter: {$and: [{"ipo.valuation_amount":{$gte:100000000}}, {founded_year:{$lt:2010}}]}
```

Comment:
I guess it's a mistake by copy pasting as it's the one as previous iteration.
Answer: {$and: [{number_of_employees:{$lt:1000}}, {founded_year:{$lt:2005}}]}